banzai_nres/tests/test_extract.py

```diff
@@ -152,7 +162,7 @@ def five_hundred_square_image(maxflux,number_traces,trace_width, read_noise=10):
     data += np.random.normal(0.0, read_noise, size=data.shape)
     uncertainty = np.sqrt(data + read_noise ** 2)
 
-    image = NRESObservationFrame([EchelleSpectralCCDData(data=data, uncertainty=uncertainty,
+    image = NRESObservationFrame([EchelleSpectralCCDData(data=data, uncertainty=uncertainty, wavelengths=10*traces,
```

Comment:
This 10 * traces seems weird.

Reply:
This is a shortcut to make it so that traces and wavelengths werent the same array for the tests. Open to suggestions if this is confusing.

Reply:
Maybe just add a comment. and use np.zeros_like just to be a little clearer that there isn't magic in the 10 value.